### DISCUSSION TITLE
Move Insulin Pens from Experimental section to "Data Sync" section.

### DISCUSSION THREAD

**Discussion by Lupum1001** [NightscoutFoundation/xDrip#2108] [Ideas]
I noticed that @jamorham recently combined the different insulin pens into a new PreferenceScreen.
<img width="300" alt="image" src="https://user-images.githubusercontent.com/52087143/167331589-d0fd58ba-202c-4aa6-b16e-83d646a75da3.png">

Do people have any objection to me moving it up to the "Data Sync" section between Glucose Meters and Smart Watch Features?

<img width="300" alt="image" src="https://user-images.githubusercontent.com/52087143/167331892-85b48f4b-f474-4278-9333-df33fa10e778.png">

I can try to come up with an alternative way of tagging them as experimental if you like (i.e. something that doesn't embed it into the translation strings directly).

**Comment by Lupum1001** [comment]
For example, I was considering something roughly along these lines.
![image](https://user-images.githubusercontent.com/52087143/167442925-3522d313-6e4f-44c6-a6be-7cabb2382c62.png)

**Comment by maxmuzy** [comment]
I think it is a great idea. Since each item will be tagged in some way, there could be an option to "Show experimental items". I'm not sure if my opinion will make much difference but I thought it was relevant to support your idea. 

Initially I have been working on translation into my language, but I intend to contribute when I understand the project a little more.
